plugins/modules/route53_info.py

```diff
@@ -630,16 +656,26 @@ def get_health_check():
 
     if module.params.get('health_check_method') == 'details':
         results = client.get_health_check(**params)
+        results["health_check"] = camel_dict_to_snake_dict(results["HealthCheck"])
+        module.deprecate(
+            "The 'CamelCase' return values with key 'HealthCheck' is deprecated \
+                and will be replaced by 'snake_case' return values with key 'health_check'. \
+                Both case values are returned for now.",
```

Comment:
There's already a note in the documentation that this is being removed in 6.0.0...

Reply:
yes, it's the same deprecation message, just moved it a bit higher in the function as part of bugfix